index.js

```diff
@@ -99,19 +99,26 @@ module.exports = (params) => {
 // Work out whether we should be disabling any assertions *now* (if we are at all)
  params.disableAssertionsNow = false;
 
+ // Work out when now is
+ const rightNow = new Date();
+
   // Check format of all ranges in the assertionDisableDuringDateRanges array of range objects, and convert month-day formats to date ranges that start within the current year
   params.assertionDisableDuringDateRanges.forEach(range => {
     if (range.fromDate instanceof Date && range.toDate instanceof Date) {
       // range is between two dates already so no further processing required
     }
     else if (Number.isInteger(range.fromMonth) && Number.isInteger(range.fromDay) && Number.isInteger(range.toMonth) && Number.isInteger(range.toDay)) {
-      range.fromDate = Date(Date.now().getFullYear, range.fromMonth, range.fromDay);
-      range.toDate = Date(Date.now().getFullYear, range.toMonth, range.toDay);
```

Comment:
I think you could do similar to above e.g.: `range.toDate = new Date(rightNow.getFullYear(), range.toMonth - 1, range.toDay);`